python/rmm/_lib/memory_resource.pyx

```diff
@@ -297,13 +297,14 @@ cpdef void set_current_device(int device) except *:
         raise RuntimeError(f"Failed to set CUDA device with error: {err}")
 
 
-cpdef _initialize(
+cpdef void _initialize(
     bool pool_allocator=False,
     bool managed_memory=False,
     object initial_pool_size=None,
     object devices=0,
     bool logging=False,
     object log_file_name=None,
+    bool avoid_initialization=False,
```

Comment:
Can we avoid the double-negative here? How about `device_initialization=True` or `do_initialize=True` instead?

Reply:
Fixed, `avoid_initialization` --> `cuda_initialization`